### ISSUE TITLE
useOpenTelemetry does not end spans after execution

### ISSUE THREAD

**Issue by galpin** [graphql-hive/envelop#1192]
**Describe the bug**

The `useOpenTelemetry` plug-in does not end tracer spans after execution. 

**To Reproduce**

```js
import { createServer } from 'graphql-yoga';
import { useOpenTelemetry } from '@envelop/opentelemetry';

const server = createServer({
  typeDefs: /* GraphQL */ `
    type Query {
      greetings: String!
    }
  `,
  resolvers: {
    Query: {
      greetings: () => 'Hello World!',
    },
  },
  plugins: [useOpenTelemetry()],
});
server.start();
```

**Expected behavior**

Execution spans to be exported to the console.

**Environment:**

- OS: Linux codespaces_8fc9e5 5.4.0-1064-azure
- `@envelop/opentelemetry`: 2.0.0
- NodeJS: v14.17.6

**Additional context**

I have fixed the issue (and added basic tests for verify only this behaviour) in [a fork here](https://github.com/galpin/envelop/commit/4b2723ce847c27a63385cc97ae8f22dd3d58b128) (I can submit a draft PR).

**Comment by galpin** [comment]
I have created a draft PR #1193 which includes the fix.

**Comment by dotansimha** [comment]
Fixed in `@envelop/opentelemetry@2.0.1` 🎉

**Comment by galpin** [comment]
Thanks @dotansimha.
